libcodechecker/libhandlers/server.py

```diff
@@ -426,7 +426,11 @@ def main(args):
         GenericSuppressHandler(None, False)
 
     if 'reset_root' in args:
-        os.remove(os.path.join(args.config_directory, 'root.user'))
+        try:
+            os.remove(os.path.join(args.config_directory, 'root.user'))
+        except OSError:
+            # File doesn't exist.
+            pass
         LOG.info("Master superuser (root) credentials invalidated and "
```

Comment:
If the root user doesn't exist, then nothing is "invalidated", because there is nothing to invalidate in the first place. I would go with moving the log line into the `try` statement too.

So if `server --reset-root` is the first start then the behaviour is the same as if `--reset-root` was not given at all.